dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -121,10 +132,12 @@
       <FunctionLevelLinking>true</FunctionLevelLinking>
     </ClCompile>
     <Link>
-      <!-- Configure the release build binary to be as required by internal API scanning tools. -->
-      <LinkIncremental>false</LinkIncremental>
-      <GenerateDebugInformation>true</GenerateDebugInformation>
-      <!-- EnableCOMDATFolding and OptimizeReferences are set to true below for all build configurations. -->
+      <!-- Note: /PROFILE is required for API scanning, and is set above for all build configurations.
+           However, this also sets /OPT:NOICF which disables EnableCOMDATFolding. For Release builds
+           it is suggested to re-enable this, but the only way to do so is by setting additional command
+           line options to the linker, after /profile is passed.
+       -->
+      <AdditionalOptions>/opt:icf %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Isn't /OPT:ICF the same as `<EnableCOMDATFolding>true</EnableCOMDATFolding>` ?

Reply:
Right, `/OPT:ICT` is the same as `EnableCOMDATFolding`, but the issue is that setting `Profile=true` actually **disables** `/OPT:ICF`, as it implies `/OPT:NOICF`.
The only way to get it back is by adding `AdditionalOptions` which get passed after `/profile`, and turn it back on.
(It kind of feels like a kludge, but this is actually what the APIScan docs suggest doing...)

Reply:
I pushed changes which no longer use `Profile`, so we don't need to turn things back on again after they have been turned off.